docs/kb/semgrep-ci/github-upload-findings-in-security-dashboard.md

```diff
@@ -0,0 +1,79 @@
+---
+description: To prevent "resource not accessible by integration" error when running job to upload findings to GitHub's Advanced Security Dashboard
+tags:
+  - Semgrep CI
+  - GitHub
+---
+
+## Why aren't findings populating in the GitHub Advanced Security Dashboard after running Semgrep in CI?
+
+When scanning with Semgrep in CI, findings automatically populate in the Semgrep Cloud Platform. To show findings in the GitHub Advanced Security Dashboard, you must first run an alternate job that uploads findings in the form of a `SARIF` file to the dashboard. See  [Sample GitHub Actions configuration file](https://semgrep.dev/docs/semgrep-ci/sample-ci-configs/#sample-github-actions-configuration-file) for an example.
```

Comment:
```suggestion
When scanning with Semgrep in CI, findings automatically populate in Semgrep Cloud Platform. To show findings in the GitHub Advanced Security Dashboard, you must first run an alternate job that uploads findings in the form of a `SARIF` file to the dashboard. See  [Sample GitHub Actions configuration file](https://semgrep.dev/docs/semgrep-ci/sample-ci-configs/#sample-github-actions-configuration-file) for an example.
```

Two threads about using the: 
* [Articles for our products](https://semgrepinc.slack.com/archives/C02JT6YA5S7/p1684444217271019)
* [A more in-depth look at "the"](https://semgrepinc.slack.com/archives/C02JT6YA5S7/p1684444774683329)